src/annotator/sidebar.coffee

```diff
@@ -3,6 +3,8 @@ raf = require('raf')
 Hammer = require('hammerjs')
 
 Host = require('./host')
```

Comment:
All the same-directory requires should be grouped together with no spaces in between.